workspaces/ui/src/contexts/MockDataContext.js

```diff
@@ -6,13 +6,14 @@ MockDataContext.displayName = 'MockDataContext';
 // only expose the Provider component, but hide the Consumer in favour of hooks
 export const { Provider } = MockDataContext;
 
-function createDebugSession({ sessionId }) {
+function createDebugSession({ sessionId, customerSession }) {
+  const collection = customerSession ? 'customer-sessions' : 'example-sessions';
```

Comment:
if we compose things as I describe in my other comments, this parameter would be applied/injected directly from outside